bootstrapvz/providers/ec2/tasks/ebs.py

```diff
@@ -16,7 +16,12 @@ def run(cls, info):
             formatted_tags = {k: v.format(**info.manifest_vars) for k, v in raw_tags.items()}
             tags = [{'Key': k, 'Value': v} for k, v in formatted_tags.items()]
 
-        info.volume.create(info._ec2['connection'], info._ec2['host']['availabilityZone'], tags)
+        # EBS volumes support encryption. KMS key id is optional and default key
+        # is used when it is not defined.
+        encrypted = info.manifest.data['provider'].get('encrypted', False)
+        kms_key_id = info.manifest.data['provider'].get('kms_key_id')
```

Comment:
This will fail if `kms_key_id` is not set, did you forget a `, None)` ?

Reply:
My understanding is that dict.get() returns None for missing objects if nothing else is defined, so I left that out on purpose. Taking out the kms_key_id from the manifest has been working for me without problems. Having the default defined as None would make it more visible, though, so adding it makes sense.

https://docs.python.org/3/library/stdtypes.html

class dict
get(key[, default])
Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a KeyError.